data/static/base_info.csv

```diff
@@ -17,6 +18,7 @@ Czech Republic - Czech Development Agency,,Government,,0
 Estonia,,Government,,0
 European Commission (EC) - DG Humanitarian Aid and Civil Protection (ECHO),2013-09-26,Other Public Sector,ec-echo,64.1
 Finland - Ministry of Foreign Affairs,2011-11-25,Government,finland_mfa,36.15
+France - Ministry for Europe and Foreign Affairs,2014-07-01,Government,france,
```

Comment:
Joined on 24th of October 2017. Baseline value should be recalculated for 1 November 2017. It has been agreed that the first of the month will be taken for any new signatories.